pkg/cmd/extension/browse/browse.go

```diff
@@ -365,14 +374,15 @@ func ExtBrowse(opts ExtBrowseOpts) error {
 	readme.SetBorder(true).SetBorderColor(tcell.ColorPurple)
 
 	help := tview.NewTextView()
-	help.SetText(
-		"/: filter  i/r: install/remove  w: open in browser  pgup/pgdn: scroll readme  q: quit")
-	help.SetTextAlign(tview.AlignCenter)
+	help.SetText("?: help  q: quit")
```

Comment:
Could this help blurb still outline the main keyboard shortcuts, so that a first-time user does not have to keep referring to `?` to learn how to use the tool? E.g.

```
j/k: move  i: install  r: remove  w: web  ?: help  q: quit
```

And for readability, could the shortcut keys be highlighted somehow? For instance, with inverted background color

Reply:
agreed, done. I used bold on the keys.